frontend_app_gallery/test/container/Gallery.spec.js

```diff
@@ -161,55 +165,43 @@ describe('<Gallery />', () => {
           wrapper.setState({ ...stateMock, displayedPictureIndex: 2 })
-          wrapper.instance().handleContentModified({ fields: { content: pictures[1] } })
+          wrapper.instance().handleContentModified({ content: pictures[1] })
           expect(wrapper.state().imagePreviewList[wrapper.state().displayedPictureIndex].label).to.equal(pictures[2].label)
 
           wrapper.setState({ ...stateMock, displayedPictureIndex: 0 })
-          wrapper.instance().handleContentModified({ fields: { content: pictures[1] } })
+          wrapper.instance().handleContentModified({ content: pictures[1] })
           expect(wrapper.state().imagePreviewList[wrapper.state().displayedPictureIndex].label).to.equal(pictures[0].label)
 
           wrapper.setState({ ...stateMock, displayedPictureIndex: 1 })
-          wrapper.instance().handleContentModified({ fields: { content: pictures[0] } })
+          wrapper.instance().handleContentModified({ content: pictures[0] })
           expect(wrapper.state().imagePreviewList[wrapper.state().displayedPictureIndex].label).to.equal(pictures[1].label)
         })
 
         it('should ignore files from other folders', () => {
           wrapper.setState(stateMock)
-          wrapper.instance().handleContentModified({ fields: { content: { ...pictures[0], label: 'NotRelevant', parent_id: folderId + 1 } } })
-          expect(wrapper.state().imagePreviewList.every(image => image.label !== 'NotRelevant')).to.equal(true)
+          wrapper.instance().handleContentModified({ content: { ...pictures[0], label: 'NotRelevant', parent_id: folderId + 1 } })
+          expect(wrapper.state().imagePreviewList.every(image => image.label !== 'NotRelevant'))
         })
 
         it('should ignore files from other workspaces', () => {
           wrapper.setState(stateMock)
-          wrapper.instance().handleContentModified({ fields: { content: { ...pictures[0], label: 'NotRelevant', workspace_id: 2 } } })
-          expect(wrapper.state().imagePreviewList.every(image => image.label !== 'NotRelevant')).to.equal(true)
-        })
-      })
-      describe('moving a content outside the current folder', () => {
-        it('should remove this content from the list when it was moved in a other workspace', () => {
-          wrapper.setState(stateMock)
-          wrapper.instance().handleContentModified({ fields: { content: { ...pictures[0], workspace_id: 2 } } })
-          expect(wrapper.state().imagePreviewList.every(image => image.filename !== pictures[0].filename)).to.equal(true)
-        })
-        it('should remove this content from the list when it was moved in a other folder and same workspace', () => {
-          wrapper.setState(stateMock)
-          wrapper.instance().handleContentModified({ fields: { content: { ...pictures[0], parent_id: folderId + 1 } } })
-          expect(wrapper.state().imagePreviewList.every(image => image.contentId !== pictures[0].content_id)).to.equal(true)
```

Comment:
Have these tests been removed?

Reply:
Err… I took the modifications from master for gallery here, not thinking that tests were added in develop. I'll add them again.

Reply:
Corrected.